feat: use canonical k8s + terraform in the get started guide (#37)
canonical/charmed-kubeflow-documentation: docs/how-to/manage/upgrade/upgrade-1.10-1.11.rst

.. _upgrade_1.10_1.11:

Upgrade from 1.10 to 1.11
=========================

This guide describes how to upgrade Charmed Kubeflow (CKF) from 1.10 to 1.11.

> Upgrading the solution by more than one minor version to another is not supported, nor is it recommended.

Reviewer: Suggestion: Wording improvements.

```suggestion
Upgrading the solution by more than one minor version at a time is not recommended and not supported.
```
